client/src/js/components/bhServiceUuidSelect.js

```diff
@@ -0,0 +1,38 @@
+angular.module('bhima.components')
+  .component('bhServiceUuidSelect', {
+    templateUrl : 'modules/templates/bhServiceUuidSelect.tmpl.html',
```

Comment:
Instead of writing a new component for this, would it be possible to just update the existing [`bhServiceSelect` component](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/js/components/bhServiceSelect.js) to take in a service uuid?

I think the best way to do this would be:
  1. Add the option to the component to pass in a `serviceUuid`. (`serviceUuid : '<?'`)
  2. Add a `$onChanges` function that will filter the services and set `serviceId` equal to the service that matches the `serviceUuid`.  Like this:
```js
$ctrl.$onChanges = (changes) => {
  if (changes && changes.serviceUuid) {
    $ctrl.services.forEach(service => {
       if (service.uuid === changes.serviceUuid) {
         $ctrl.serviceId = service.id;
       }
    })
  }
};
```